torch/onnx/utils.py

```diff
@@ -230,7 +230,7 @@ def _trace_and_get_graph_from_model(model, args, training):
     # can turn training=True (or None, to preserve whatever the original
     # training mode was.)
     with set_training(model, training):
-        trace_graph, torch_out, inputs_states = torch.jit._get_trace_graph(model, args, _force_outplace=True, _return_inputs_states=True)
+        trace_graph, torch_out, inputs_states = torch.jit._get_trace_graph(model, args, _force_outplace=False, _return_inputs_states=True)
```

Comment:
@bowenbao - have you seen any existing tests failures with this change? If not, I think we can put this change in, and track if any other scenarios in ONNX export break.

Reply:
So far no test failures. https://github.com/pytorch/pytorch/blob/3657df383669a651a2d7717f93e59fbd9a8458c4/torch/onnx/utils.py#L618
This part handles what the `force_outplace` flag is doing as well.